laletterbuilder/schema.py

```diff
@@ -74,7 +79,7 @@ def perform_mutate(cls, form, info: ResolveInfo):
 
 
 @schema_registry.register_mutation
-class LandlordNameAddressEmail(OneToOneUserModelFormMutation):
+class LandlordNameAddress(OneToOneUserModelFormMutation):
```

Comment:
FYI if we had already deployed this app and people were using it in production, there'd be a risk in changing the mutation that people using a cached version of the frontend would get an error when their old frontend tried to send the old version of the mutation to the new version of the backend. Normally we'd mitigate this by leaving `LandlordNameAddressEmail` as is and writing a *new* mutation called `LandlordNameAddress` or `LandlordNameAddressv2` or something. Then, after several weeks & all caches expired, we'd deprecate the old mutation by deleting the old `LandlordNameAddressEmail` code. You'll see several places where we have both an `xMutation` and an `xMutationv2` - that's why :)

Since we don't have any users tho, we can change as much as we want to without worrying about people having mismatched versions of FE/BE.